lib/Github/Client.php

```diff
@@ -123,7 +123,7 @@ class Client
      */
     public function __construct(Builder $httpClientBuilder = null, $apiVersion = null, $enterpriseUrl = null)
     {
-        $this->responseHistory = new History();
+        $this->responseHistory = interface_exists('Http\Client\Common\HttpMethodsClientInterface') ? new Httplug2History() : new History();
```

Comment:
You can just do `HttpMethodsClientInterface::class`. Importing a class that doesn't exist won't crash the runtime. It is purely just binding the name. No class loading will go on.

Reply:
yes, you are right!